project/sites/ircommission/themes/ircommission_theme/src/scss/components.scss

```diff
@@ -563,6 +559,7 @@ footer {
 
 .block-views-exposed-filter-blockpublications-search-publication-search-page {
   margin-top: 4.5rem;
```

Comment:
Unity theme should take care of exposed filter styling, you shouldn't need to change it.

Reply:
Thanks I have now removed this margin